packages/dso-toolkit/src/components/document-component-list/document-component-list.scss

```diff
@@ -0,0 +1,12 @@
+@use "../../variables/units";
+@use "../list";
+
+ul.dso-document-component-list {
+  @include list.unstyled();
+
+  margin: 0;
+
+  > li + li {
```

Comment:
Deze code is copy/pasted uit `document-component.scss`, maar deze regel code was eerst `li + li {`.

Met de direct child selector wordt alleen de `li` horende bij Document Component List (`ul.dso-document-list`) gestijld.